### PR TITLE
TOMEE-2563- Translate to portuguese-bean-validation-design-by-contract

### PR DESCRIPTION
**Pull request by Daniel-Dos**
Link to Jira : https://issues.apache.org/jira/browse/TOMEE-2563

### CODE REVIEW

**Comment by Daniel-Dos** [comment]
thanks @davidsalter : )

**Comment by jgallimore** [comment]
I'm happy to get this merged in, but if we have any Portuguese speakers (@alexferreiradev maybe?) who can give a quick review, that would be great.

File: examples/bean-validation-design-by-contract/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+Algumas implementações desta JSR implementam este apêndice (Apache bval,
+Hibernate validator por exemplo).
+
+O OpenEJB fornece um interceptador na qual permite você usar esta feature para
+fazer o design por contrato.
+
+== Design por contrato
+
+O objetivo é ser capaz de configurar com um grão mais fino seu contrato. No
+exemplo você especificar os centímetros mínimos de um homem do esporte deve pular
```
Comment (gabrielferreirapro) [inline examples/bean-validation-design-by-contract/README_pt.adoc]: "você especificar"

File: examples/bean-validation-design-by-contract/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,225 @@
+:index-group: Other Features
+:jbake-type: page
+:jbake-status: status=published
+= Design Bean Validation por Contrato
+
+== Bean Validation - Design por Contrato
+
+O Bean Validation (também conhecida como JSR 303) contém um apêndice opcional
+método de validação.
```
Comment (gabrielferreirapro) [inline examples/bean-validation-design-by-contract/README_pt.adoc]: "um apêndice opcional método de validação" soa estranho/errado para mim. Talvez "um apendice opcional que lida com validação de método" ?

Comment (Daniel-Dos) [reply examples/bean-validation-design-by-contract/README_pt.adoc]: isso ficou bom . obrigado .

Comment (Daniel-Dos) [reply examples/bean-validation-design-by-contract/README_pt.adoc]: done

File: examples/bean-validation-design-by-contract/README_pt.adoc (hunk 1 above)
Comment (Daniel-Dos) [reply examples/bean-validation-design-by-contract/README_pt.adoc]: done

**Pull request merged**